ank/doc/swdesign/README.md

```diff
@@ -529,6 +529,223 @@ Needs:
 - impl
 - utest
 
+### `ank apply [-d] [--agent agent_name] <manifest.yaml> ...`
+
+#### Ankaios manifest
+
+The Ankaios manifest is a YAML (or a JSON) file composed of a list of workload specifications under the `workloads` keyword.
+
+```yaml
+# Example of a list of two workload specifications with the names 'nginx' and 'hello1'.
+workloads:
+  nginx:
+    agent: agent_A
+    tags:
+      - key: owner
+        value: Ankaios team
+    dependencies: {}
+    updateStrategy: AT_MOST_ONCE
+    restart: true
+    accessRights:
+      allow: []
+      deny: []
+    runtime: podman
+    runtimeConfig: |
+      image: docker.io/nginx:latest
+      commandOptions: ["-p", "8081:80"]
+  hello1:
+    # For this workload the following are not set:
+    # - agent name
+    # - dependencies -> defaults to {}
+    tags: []
+    restart: true
+    updateStrategy: AT_MOST_ONCE
+    accessRights:
+      allow: []
+      deny: []
+    runtime: podman-kube
+    runtimeConfig: |
+      image: alpine:latest
+      commandOptions: [ "--rm"]
+      commandArgs: [ "echo", "Hello Ankaios"]
+```
+
+##### Workload specification
+
+A workload specification consists of the following properties:
+
+| Property | Description | Value | Is required? |
+|----------|:-----------:|:------|:----------:|
+| **workload name** (_as field key_) | It specifies an unique workload name to identify the workload in the Ankaios system. | A string of any characters (if properly quoted by either single quotes 'example' or double quotes "example"). | true |
+| **agent** | It specifies the name of the owning agent which is going to execute the workload. | A string of any characters (if properly quoted by either single quotes 'example' or double quotes "example"). |   false |
+| **tags** | It specifies a list use defined key/value objects. | A list of { `key`: some_string, `value`: some_string} or empty| true |
+| **dependencies** | It specifies inter workload dependencies. | Not specified yet and shall be set to empty.| false |
+| **updateStrategy** | It specifies the update strategy. | One of: `UNSPECIFIED`, `AT_LEAST_ONCE`, `AT_MOST_ONCE` | true |
+| **restart** | It specifies whether the workload shall be restarted when it exits | One of: `true`,`false` | true |
+| **accessRights** | It specifies lists of access rules fpr `allow` and `deny`. | Not fully specified yet and shall be set to empty list for both. | true |
+| **runtime** | It specifies the type of the runtime. | One of: `podman`, `podman-kube`. | true |
+| **runtimeConfig** | It specifies the configuration for the runtime whose configuration structure is specific for each runtime as a _string_ | As a _string_ from one of: [PodmanRuntimeConfig](#podmanruntimeconfig), [PodmanKubeRuntimeConfig](#podmankuberuntimeconfig),  | true |
+
+###### PodmanRuntimeConfig
```

Comment:
To avoid drift of configurations, maybe we can only state that a manifest is the `workloads` part of the state with an option specification of an `agent`.